cartoframes/data/utils.py

```diff
@@ -84,7 +93,17 @@ def _warn_new_geometry_column(df):
 
 
 def _compute_geometry_from_geom(geom):
-    return geom.apply(decode_geometry)
+    first_geom = next(item for item in geom if item is not None)
```

Comment:
what is this for?

Reply:
To get the first not null element from the list

Reply:
Oh OK, it wasn't clear to me, the parameter was a list.